src/main/kotlin/com/openlattice/organizations/DatasetController.kt

```diff
@@ -108,9 +111,9 @@ class DatasetController : DatasetApi, AuthorizingComponent {
     override fun getExternalDatabaseTable(
             @PathVariable(ID) organizationId: UUID,
             @PathVariable(TABLE_NAME) tableName: String): OrganizationExternalDatabaseTable {
-        val tableId = getExternalDatabaseObjectId(organizationId, tableName)
-        ensureReadAccess(AclKey(organizationId, tableId))
-        return edms.getOrganizationExternalDatabaseTable(tableId)
+        val tableFqnToId = getExternalDatabaseObjectFqnToIdPair(organizationId, tableName)
```

Comment:
Since you are using only the id there you could destructure it like that:
`val (_, id) = getExternalDatabaseObjectFqnToIdPair(organizationId, tableName)`

Reply:
oh great point!